packages/fluentui/docs/src/examples/components/Form/Usage/FormExampleErrorAndSuccessful.shorthand.tsx

```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
-import { Form, Button, Input } from '@fluentui/react-northstar';
-import { PresenceAvailableIcon } from '@fluentui/react-icons-northstar';
+import { Form, Button } from '@fluentui/react-northstar';
 
 const fields = [
   {
```

Comment:
![image](https://user-images.githubusercontent.com/4512430/84511176-6ff43980-acc6-11ea-8900-dd75fc9c2a51.png)
Would suggest changing the error here to something like "You cannot fix this error", or handle somehow inside the component's state when this should go away.

Reply:
Gonna change it to `You can't fix this error` we gonna handle the change probably in the prototype with some form lib

Reply:
Changed